language/tools/test_generation/src/bytecode_generator.rs

```diff
@@ -487,7 +512,10 @@ impl BytecodeGenerator {
         signature: &FunctionSignature,
         module: CompiledModuleMut,
     ) -> Vec<Bytecode> {
-        let number_of_blocks = 3;
+        let number_of_blocks = self.rng.gen_range(1, MAX_CFG_BLOCKS + 1);
+        // The number of basic blocks must be at least one based on the
+        // generation range.
+        assume!(number_of_blocks > 0);
```

Comment:
At least make an issue for MIRAI to add a prebuilt summary for gen_range.

Reply:
Working on a PR for MIRAI.